### ISSUE TITLE
Feature: Vehicle state color on device list

### ISSUE THREAD

**Issue by gpproton** [vitalidze/traccar-web#947]
Hi @vitalidze 

Currently in Traccar-web UI MOD vehicle state either online, moving, offline etc can only be know when mouse is hovered on it, in my opinion i think it will be much better if this various state are displayed are displayed as colored dot or circles before the devices name, i know it will be useful, what do you think?

This is an illustration below:

![state icon-colour](https://cloud.githubusercontent.com/assets/11395006/20224273/fa2e6300-a83d-11e6-9609-6b341ac640dd.png)

**Comment by vitalidze** [comment]
I agree that it could be useful.

**Comment by gpproton** [comment]
Hi @vitalidze can you give any advice on adding a little device state icon in front of each vehicle, with my guess it would probably be in src/main/java/org/traccar/web/client/view/DeviceView.java and src/main/java/org/traccar/web/client/view/DeviceView.ui.xml
i will appreciate any advice.

**Comment by vitalidze** [comment]
You are right, you need to update TreeGrid in `DeviceView` to show necessary data.

**Comment by gpproton** [comment]
Ok thanks @vitalidze i'll try if something can be done.
